src/core/DatePicker/DatePicker.tsx

```diff
@@ -124,7 +125,13 @@ export type DatePickerProps = {
    * @default false
    */
   setFocus?: boolean;
-} & Omit<CommonProps, 'title'>;
+  /**
+   * Show time picker near the calendar.
+   * @default false
+   */
+  showTime?: boolean;
+} & Omit<CommonProps, 'title'> &
+  Omit<TimePickerProps, 'date' | 'onChange'>;
```

Comment:
Omit `setFocusHour`

Reply:
YOu think there wont be a need to focus hour before day?

Reply:
omited..